data-raw/sample_books.R

```diff
@@ -0,0 +1,12 @@
+## Datasets to use in examples, tests, and vignettes.
```

Comment:
For scripts used to generate data for packages, I've found it helpful on many occasions to include a shebang:
```suggestion
#!/usr/bin/env Rscript

## Datasets to use in examples, tests, and vignettes.
```
Tends to be most helpful if you have some automated process to run them and commit dataset changes, not sure if if it's necessary here though.

Reply:
Interesting! I don't have a specific use for this to be executable on Unix right now, but I'll commit this in case. Thanks for the tip!